executor/show_stats.go

```diff
@@ -221,3 +221,25 @@ func (e *ShowExec) appendTableForStatsHealthy(dbName, tblName, partitionName str
 		healthy,
 	})
 }
+
+func (e *ShowExec) fetchShowAnalyzeStatus() {
+	for _, job := range statistics.GetAllAnalyzeJobs() {
```

Comment:
can we only show the current processing analyze jobs? when we found that the result is empty we can simply get the information that all the analyze jobs are finished?

To retrieve history jobs, I think we can use a separate command, like `show history analyze jobs`.

Reply:
The result presents more info than finished. Also, it would be hard to write tests...